docs/help-center/using-famedly/chat-backup-verification.mdx

```diff
@@ -0,0 +1,120 @@
+---
+sidebar_label: Chat Backup Verification
+title: Chat Backup Verification
+hide_title: true
+displayed_sidebar: helpcenterSidebar
+sidebar_position: 6
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+
+<div class="hero hero--primary">
+  <div class="container">
+    <h2 class="hero__title">Chat Backup Verification</h2>
+    <p class="hero__subtitle">How to use QR-Verification and what to do if your device can't scan the code or no other devices are available.</p>
+  </div>
+</div>
+
+
+## QR Verification
+
+You will get redirected to the QR-Verification page automatically in the login-flow, if at least one of the devices you want to use for verification is a mobile device.  
+If you nevertheless get redirected to the number- or emoji-verification instead, please check if there is an update for one of your devices. 
```

Comment:
What do you mean update? Following this sentence, I would open the Playstore and try to update my app, is that what is meant?

Reply:
I already thought about removing this sentence. We added it when we switched from emoji verification to number verification because some people might not have updated their phones back then.
This is just shown as new content because I changed the name of the file.

Reply:
Removed that sentence